### PR TITLE
Create organization schema

### PR DESCRIPTION
**Pull request by skoehnen**
Related to #84

### CODE REVIEW

File: v3.0/products/organization.schema.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "description": "Metadata node type (schema).",
+      "type": "string",
+      "const": "https://openminds.ebrains.eu/core/datasetVersion"
+    },
+    "@id": {
+      "description": "Metadata node identifier.",
+      "type": "string"
+    },
+    "digitalIdentifier": {
+      "description": "???",
```
Comment (skoehnen) [inline v3.0/products/organization.schema.json]: I am not sure what a good description would be here.
@UlrikeS91 @lzehl maybe you have some good suggestions.

What would the digitalIdentifier of an organization be?

**Review by skoehnen — COMMENTED** [review]
The required property missing

File: v3.0/products/organization.schema.json (hunk 1 above)
Comment (apdavison) [reply v3.0/products/organization.schema.json]: Maybe take a look at https://ror.org and https://www.grid.ac ?

Comment (apdavison) [reply v3.0/products/organization.schema.json]: ROR (e.g. https://ror.org/02nv7yv05) looks like the best. More information at https://scholarlykitchen.sspnet.org/2019/12/04/are-you-ready-to-ror-an-inside-look-at-this-new-organization-identifier-registry/

Comment (skoehnen) [reply v3.0/products/organization.schema.json]: Ah interesting.
I wasn't aware of those registrars

But that makes sense to me.

File: v3.0/products/organization.schema.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "$id": "https://openminds.ebrains.eu/core/organization.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Identify organizations that were involved in producing the research products or their versions.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "@type": {
+      "description": "Metadata node type (schema).",
+      "type": "string",
+      "const": "https://openminds.ebrains.eu/core/datasetVersion"
```
Comment (lzehl) [inline v3.0/products/organization.schema.json]: this needs to be ".../core/organization"

File: v3.0/products/organization.schema.json (hunk 1 above)
Comment (lzehl) [inline v3.0/products/organization.schema.json]: Globally unique and (best) persistent digital identifier for the organization.

Comment (lzehl) [reply v3.0/products/organization.schema.json]: I agree with  @apdavison here, ROR should be preferred but we will not limit it to this

**Review by lzehl — COMMENTED** [review]
Required properties are okay for now. The globalIdentifier should be highly recommended, though.

**Review by lzehl — CHANGES_REQUESTED** [review]
- `@type` needs to be corrected
- `globalIdentifier` `description` : I made a different suggestion

**Comment by lzehl** [comment]
@skoehnen I think Ulrike's review is not needed in addition. @UlrikeS91 hope you agree.

**Comment by skoehnen** [comment]
> @skoehnen I think Ulrike's review is not needed in addition. @UlrikeS91 hope you agree.

Hi, I added both of you so you know that someone can review it.
When we started with this we agreed that it is okay if one of us reviews the pull requests and I was still working under this assumption.
If that changed in the meantime, I will change how I set up the Pull Requests?

File: v3.0/products/organization.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/organization.schema.json]: > Globally unique and (best) persistent digital identifier for the organization.

What do you mean with best in this case?
The best identifier or the identifier should better be persistent.

In either case I would remove `(best)`

File: v3.0/products/organization.schema.json (hunk 2 above)
Comment (skoehnen) [reply v3.0/products/organization.schema.json]: Oh yes, you are totally correct.

Comment (skoehnen) [reply v3.0/products/organization.schema.json]: Is fixed.

**Comment by skoehnen** [comment]
> Required properties are okay for now. The globalIdentifier should be highly recommended, though.

Yes I agree, sadly we only have required 😅 
And I think to require to would maybe a bit too strict and cause problems.

**Comment by skoehnen** [comment]
> * `globalIdentifier` `description` : I made a different suggestion

You called the digitalIdentifier globalIdentifier a few times now.
Should we change the name.

(I had to check #84 to make sure I didn't make an error in the first place 😅)

File: v3.0/products/organization.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/organization.schema.json]: @lzehl I updated the description if you want to take a look.

**Comment by lzehl** [comment]
@skoehnen so sorry :see_no_evil: the property is still called `digitalIdentifier`... I just always confuse it... In principle they identifiers should be all globally unique, so we could also call the property `globalIdentifier` but I wasn't sure if "global" is really always true for all the cases this property (and corresponding schema) is used.

**Comment by skoehnen** [comment]
> @skoehnen so sorry 🙈 the property is still called `digitalIdentifier`... I just always confuse it... In principle they identifiers should be all globally unique, so we could also call the property `globalIdentifier` but I wasn't sure if "global" is really always true for all the cases this property (and corresponding schema) is used.

Okay 👍
Just wanted to make sure

**Review by UlrikeS91 — APPROVED** [review]

**Comment by UlrikeS91** [comment]
@skoehnen and @lzehl, Is there anything I need to do here? Based on the comments:

1) Yes, I agree. Only one review should be enough. So, whenever Lyuba reviews, I don't need to review as well. I also got it like Stefan explained it, and if that changed, I also didn't get that message, @skoehnen :D

2) I reviewed it anyway to see if something was missing (which would explain why the schema isn't merge into the master yet), but it looks good to me. Stefan change the description as requested by Lyuba. @lzehl, Do you agree that we should merge this now?

**Comment by lzehl** [comment]
@UlrikeS91 yes go ahead and merge it :+1:

**Review by lzehl — APPROVED** [review]

**Pull request merged**